generators/app/templates/src/app/home/__bootstrap._home.component.html

```diff
@@ -6,5 +6,16 @@ <h1>
     </h1>
     <app-loader [isLoading]="isLoading"></app-loader>
     <q [hidden]="isLoading">{{quote}}</q>
+<% if (props.angulartics) { -%>
+    <p>
+      <button
+        class="btn"
```

Comment:
It's missing a button style, like `btn-primary` (see https://getbootstrap.com/docs/4.0/components/buttons/)